modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,260 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// https://github.com/tektoncd/hub/pull/547
+// Shiv Pratap Verma @pratap0007
+
+[id="pac-new-features-1-8_{context}"]
+=== {pac}
+// {pac} owners: Shivam Mukhade Savita Ashture
+
+:FeatureName: {pac} (PAC)
+include::snippets/technology-preview.adoc[]
+
+* {pipelines-title} 1.7.x shipped with {pac} 0.5.x. The current update ships with {pac} 0.10.x. This change creates a new route in the `openshift-pipelines` namespace for the new controller. You must update this route in GitHub Apps or webhooks that use {pac}. To fetch the route, use the following command:
++
+[source,terminal]
+----
+oc get -n openshift-pipelines pipelines-as-code-controller  --template='https://{{ .spec.host }}'
+----
+// no link
+
+* With this update, {pac} supports GitLab integration with features such as the following:
+** ACL (Access Control List) on project or group
+** `/ok-to-test` support from allowed users
+** `/retest` support.
+//For more information, https://pkg.go.dev/github.com/openshift-pipelines/pipelines-as-code/pkg/provider/gitlab#section-readme
+// no link
+
+* With this update, you can perform advanced pipeline filtering with Common Expression Language (CEL). With CEL, you can match pipeline runs with different Git provider events through special annotations in the `PipelineRun` resource. For example:
++
+[source,yaml]
+----
+  ...
+  annotations:
+     pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" && target_branch == "main" && source_branch == "wip"
+----
+// no link
+
+* With this update, {pac} supports multiple pipeline runs on the same event. The pipeline runs operate in parallel and report back to the Git provider interface.
+// no link
+
+* With this update, you can rerun a pipeline run by commenting `/test` or `/retest` on a pull request. You can also specify the pipeline run by name. For example, by entering `/test <pipelinerun_name>` or  `/retest <pipelinerun-name>`
+// no link
+
+* This update adds a new webhook validator to ensure only one {pac} repository is created by one git repository URL.
+// no link
+
+* With this update, you can delete a repository custom resource and its associated secrets by using the new `tkn-pac repo delete` command.
+// no link
+
+* With this update, you can use the new `tkn-pac setup-webhook cli` command to set up a webhook with a repository hosted by a Git provider such as GitHub, GitLab, or Bitbucket Cloud.
+// no link
+
+[id="chains-new-features-1-8_{context}"]
+=== Chains
+
+:FeatureName: Tekton Chains
 include::snippets/technology-preview.adoc[]
 
+* With this update, you can log in to a vault by using OIDC instead of a static token. This change means that Spire can generate the OIDC credential so that only trusted workloads are allowed to log in to the vault. Additionally, you can pass the vault address as a configuration value rather than inject it as an environment variable.
```

Comment:
```suggestion
* With this update, you can log in to a vault by using OIDC rather than a static token. This change means that Spire can generate the OIDC credential so that only trusted workloads are allowed to log in to the vault. Additionally, you can pass the vault address as a configuration value rather than inject it as an environment variable.
```
Per Vale - Use `rather than` rather than `instead of`